exercises/06-redux-saga/src/modules/users/users-selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { createSelector } from 'reselect'
+
+import { RootState } from 'modules/root/root-reducer'
+import { User } from './user-types'
+
+const getState = (state: RootState) => state.users
+
+export const getTitle = createSelector(getState, (state) => state.title)
+
+const getUsers = createSelector(getState, (state) => state.users)
+
+const withUppercaseLastName = (user: User) => ({
```

Comment:
A nitpick but maybe `withUpperCaseLastName` with `C` because it's `.toUpperCase`? Not sure. You don't have to change it.

Reply:
Fixed, see 664df42fb4cff222735b029283a7bf593d769621